test/e2e/helpers.go

```diff
@@ -675,18 +675,18 @@ func resolveKubetestRepoListPath(version string, path string) (string, error) {
 		return "", err
 	}
 
-	v125, err := semver.Make("1.25.0-alpha.0.0")
+	v124, err := semver.Make("1.24.0-alpha.0.0")
 	if err != nil {
```

Comment:
comment on line 659 also refers to v1.25+ and should get updated

Reply:
updated!